arm/Microsoft.SignalRService/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,61 @@
+@sys.description('Required. The IDs of the principals to assign the role to.')
+param principalIds array
+
+@sys.description('Required. The name of the role to assign. If it cannot be found you can specify the role definition ID instead.')
```

Comment:
Also, please add a parameter file & test run/badge. If that's not possible on your end, let me know, then I'll try to set something up on our side

Reply:
Added tests in .test folder